tests/PHPSA/ApplicationTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\PHPSA;
+
+use PHPSA\Application;
+
+class ApplicationTest extends TestCase
+{
+    /**
+     * @covers \PHPSA\Application::__construct
```

Comment:
I know what it does, I'm just not sure that we actually want to use this feature :)